examples/MagicWeather/app/build.gradle

```diff
@@ -1,20 +1,12 @@
-plugins {
-    id 'com.android.application'
-    id 'kotlin-android'
-}
+apply from: "$rootProject.projectDir/base-application.gradle"
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion "30.0.3"
-
     defaultConfig {
         applicationId "com.revenuecat.purchases_sample"
         minSdkVersion 26
```

Comment:
is this a diff minSdk for a reason?

Reply:
I don't know, I think it has a dependency that requires 26, but I am not sure